docs/source/importing_java_sdk.rst

```diff
@@ -0,0 +1,52 @@
+========================
+Importing the Java SDK
+========================
+
+.. note::
+   The Sawtooth Java SDK requires JDK 8 or above.
+
+Once you've got a working version of Sawtooth, there are a few additional
+steps you'll need to take to get started developing for Sawtooth.
+
+**Java Project**
```

Comment:
There are a few Sphinx extensions that allow you to have tabbed views. That might be nice for these places where we have a Java project/Android project, or Java/Kotlin examples. Not sure if it is worth the complication of adding a 3rd party extension though. Maybe run it by @chenette

Reply:
I tried to use this one [sphinxcontrib-osexample](https://github.com/svx/sphinxcontrib-osexample) for codeblock tabs. But I couldn't make it work following their documentation. So, I decided not to spend much time trying to figure it out. But I can spend more time in it, if needed.

Reply:
Tabbed view isn't going to work great as a PDF (or other non-interactive formats), so better to avoid.